openspec/changes/archive/2026-06-07-model-switching/tasks.md

```
@@ -0,0 +1,85 @@
+## 1. Data Model and Types
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)